subscription-engine/kafka-ws/bin/entryPoint.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+# -e Exit imediatly if a comand exit with non-zero status
+# -x Print commands and their arguments as they are executed
+
+# Debug mode
+if [ ! -z "${DEBUG+x}" ]; then
+    set -x
+fi
+
+export KAKKA_WS_CONNECTION_RETRY_COUNT=${KAFKA_WS_APP_CONNECTION_RETRY_COUNT:-"5"}
```

Comment:
Why is it necessary to use `export` here?

Reply:
KAKKA -> KAFKA

Reply:
does it make sense for this variable to be called that? The others do not seem to have this same pattern:
> KAFKA_WS_**APP**_CONNECTION_RETRY_COUNT

Reply:
> does it make sense for this variable to be called that?
> 
> > KAFKA_WS_**APP**_CONNECTION_RETRY_COUNT
> 
> The others do not seem to have this same pattern...

This pattern is the configManager pattern,m we don't want it to differ from development, I updated it.